src/Middleware/RateLimiting/src/DefaultAcquiredLease.cs

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Threading.RateLimiting;
+
+namespace Microsoft.AspNetCore.RateLimiting;
+
+internal sealed class DefaultAcquiredLease : RateLimitLease
+{
+    public override bool IsAcquired => true;
```

Comment:
Wouldn't it make more sense to return an unacquired lease? Don't we risk bad actors canceling requests while they're in the queue, the middleware continuing with an acquired lease, and then completely blowing past whatever limit was in place to protect the backend or db?

I wouldn't expect the inner limiter(s) to account for canceled attempts to acquire towards resource utilization. All of our default limiters would treat canceled requests as unacquired, right?